redeem/EndStop.py

```diff
@@ -108,11 +93,13 @@ def read_value(self):
 
   def callback(self):
     """ An endStop has been hit """
-    logging.info("End Stop " + self.name + " hit!")
+    type_string = "hit" if self.hit else "released"
+    event_string = "End Stop {} {}!".format(self.name, type_string)
+    logging.info(event_string)
     if "toggle" in self.printer.comms:
```

Comment:
Suggest combining the following to avoid repetition:
+    for channel in ["octoprint", "toggle"]:
+      if channel in self.printer.comms:
+        self.printer.comms[channel].send_message(event_string)